### ISSUE TITLE
missing aesthetics documentation

### ISSUE THREAD

**Issue by rrrlw** [tdaverse/ggtda#38]
see convo in #21

**Comment by corybrunson** [comment]
This might technically be a separate issue, but it's so closely related that i think it belongs here. As i explained to myself [here](https://corybrunson.github.io/2020/04/17/calculate-aesthetics/), documentation is especially important for _computed variables_ used for _calculated aesthetics_. **ggtda** includes one computed variable, `persistence`, computed by `stat_persistence()`. Could we include a section for it in this stat's documentation? (Note: It's already used in one example, which is good.)

**Comment by corybrunson** [comment]
This is addressed in d0a4ec29293755f1f958f623f36b0be963dcb773 and can be closed once that commit is merged to `main`.
